docs/inference-with-tvm.md

```diff
@@ -0,0 +1,101 @@
+---
+permalink: /docs/inference-with-tvm
+---
+
+# Inference Task with TVM in Teaclave
+
+Inference and model training are two important topics in machine learning.
+Thanks to [TVM](https://tvm.apache.org/) and [WebAssembly
+Executor](https://teaclave.apache.org/docs/executing-wasm/), Teaclave is now
+able to run the former—inference tasks. TVM can convert a model (or computation
+graph) to an intermediate representation (IR) defined by TVM, and compile the
+binary of this model from the IR. Since TVM recruits LLVM to emit binary code
+and LLVM support WebAssembly as backend, Teaclave's WebAssembly Executor can
+then execute the model's binary with additional lightweight runtime provided by
+TVM. 
+
+Although TVM has already provided an [wasm-standalone example
+app](https://github.com/apache/tvm/tree/main/apps/wasm-standalone), we still
+cannot copy and run it in Teaclave due to lack of WASI support and specific
+context file interface. This document mainly focuses on the *what's different*
+in Teaclave and we will finally build a MNIST inference function for Teaclave.
```

Comment:
I don't think this paragraph is useful. Note that this is the long-lasting document but not a blog.

Reply:
We can still keep it.